voila/configuration.py

```diff
@@ -140,3 +140,19 @@ class VoilaConfiguration(traitlets.config.Configurable):
         help="""Size of pre-heated kernel pool for each notebook. Zero or negative number means disabled.
         """,
     )
+
+    extension_whitelist = List(
```

Comment:
Maybe `0.5.0` could be a good time to switch to `allowlist` and `denylist`?

Since it was also suggested in https://github.com/voila-dashboards/voila/pull/725.